api/ragflow_server.py

```diff
@@ -38,9 +38,9 @@
 from common.file_utils import get_project_base_directory
 from common import settings
 from api.db.db_models import init_database_tables as init_web_db
-from api.db.init_data import init_web_data
-from common.versions import get_ragflow_version
-from common.config_utils import show_configs
+from api.db.init_data import init_web_data, init_superuser
+from api.versions import get_ragflow_version
+from api.utils.configs import show_configs
```

Comment:
The get_ragflow_version and show configs have been moved to common modules. Use following imports.

```
from common.versions import get_ragflow_version
from common.config_utils import show_configs
```